### ISSUE TITLE
Topicons integrations

### ISSUE THREAD

**Issue by KarunaGovind** [spin83/multi-monitors-add-on#39]
Any chance you could make this work with topcions (so that the icons show on all screens)?

https://extensions.gnome.org/extension/495/topicons/

Thanks.

**Comment by jonnjonzzn** [comment]
I'm actually using this extension for the purpose:

https://extensions.gnome.org/extension/1031/topicons/

Mike

**Comment by jonnjonzzn** [comment]
I assume you want to see the "activity tray" on all monitors as opposed to just the primary monitor?  I'm not sure I'd want that as a default.  The rest of the panel items are only on primary monitor now so this would appear to be a bigger project than just the Top Icons extension.

**Comment by KarunaGovind** [comment]
Yeah that's what wanted. Previously when I had 2 screens it was fine but now that I have 3, it takes me a while to notice blinking icons such as that of Pidgin.  

Fair enough if it's too big of a project, you can close the ticket. Thanks.
